src/scripts/main.js

```diff
@@ -1,5 +1,37 @@
 'use strict';
 
-document.addEventListener('click', e => {
+const wall = document.querySelector('.wall');
+const spider = wall.querySelector('.spider');
+
+wall.addEventListener('click', e => {
   // write code here
+  const wallPosition = wall.getBoundingClientRect();
+
+  const wallX = wallPosition.x + wall.clientLeft;
+  const wallY = wallPosition.y + wall.clientTop;
+
+  let x = e.clientX - wallX - spider.offsetWidth / 2;
+  let y = e.clientY - wallY - spider.offsetHeight / 2;
```

Comment:
якщо тут використати e.offsetX і e.offsetY тоді можна не створювати змінні wallX і wallY
```suggestion
  let x = e.offsetX - spider.offsetWidth / 2;
  let y = e.offsetY - spider.offsetHeight / 2;
```